rdm/cli.py

```diff
@@ -25,13 +25,36 @@ def cli(raw_arguments):
         init(args.output)
     elif args.command == 'pull':
         pull_requirements_and_reports(args.system_yml)
+    elif args.command == 'hooks':
+        install_hooks(args.dest)
 
 
 def init(output_directory):
     init_directory = pkg_resources.resource_filename(__name__, 'init')
     shutil.copytree(init_directory, output_directory)
 
 
+def install_hooks(dest=None):
+    hooks_source = pkg_resources.resource_filename(__name__, 'hooks')
+    if dest is None:
+        root = subprocess.check_output(['git', 'rev-parse', '--show-toplevel'])
+        root_str = root.strip().decode('ascii')
+        dest = root_str + '/.git/hooks'
+    copyhooks(hooks_source, dest)
+    print('Successfully installed hooks in {}'.format(dest))
+
+
+def copyhooks(source, dest):
+    if not os.path.exists(dest):
+        os.makedirs(dest)
+    for item in os.listdir(source):
+        src = os.path.join(source, item)
+        dst = os.path.join(dest, item)
```

Comment:
I think it is bad to use `dest` and `dst` in the same function.  Maybe find better names for both `src` and `dst`.  E.g. `item_src` and `item_dest`?

Reply:
I went ahead and renamed `src` and `dst` to `item_source` and `item_dest` and also changed `source` and `dest` to `dir_source` and `dir_dest` for clarity.